### DISCUSSION TITLE
No Graphic interface Windows 10

### DISCUSSION THREAD

**Discussion by Braymanlw** [jamulussoftware/jamulus#2117] [Technical Support & Help]
I have recently installed Jamulus . When I run it it runs but there is no graphic interface i.e. nothing on the screen and it does not show on the Task Bar but is running in Task Manager. Tried running it from a flash drive which works on a laptop but with same result. Running Windows 10
Regards,
Louis Whelan

**Comment by henkdegroot** [comment]
That's really odd. Have you had dual monitors setup in the past and perhaps you are using just one now? 
I did not check, but have seen other application that remembered the location at the second monitor causing the GUI no to be visible with a single monitor.

Can you also confirm the version of Jamulus you are using?

**Reply by Braymanlw** [reply]
Thanks for your prompt reply.  Windows 10 auto detects multiple displays and there are none.  Version 3.8.1 used to work on this machine but hasn't since I had a new c. drive installed. I have tried three different versions 3.6.2, 3.8.0 and 3.8.1.  They all work on two other laptops but not on this desk top.  This is the only application that doesn't work and I think the fact that it appears to be running in Task Manager but not displaying on the Task Bar has to be significant I just can't figure out why.

**Reply by henkdegroot** [reply]
Perhaps this is caused by a vc runtime library issue. Usually you would see some errors, but maybe for some odd reason you don't.
You can download the required runtime here: [https://aka.ms/vs/16/release/vc_redist.x64.exe](https://aka.ms/vs/16/release/vc_redist.x64.exe) 
You may need to remove the existing installed version first. See also this issue #1843

**Reply by pljones** [reply]
Here's a "deep clean" guide...

* Open Windows Explorer and enter `%APPDATA%` in the location bar.
* Does a `Jamulus` folder exist inside?
* What, if any, any content is in the `Jamulus` folder?
* If you go into Windows "Add or Remove programs", is Jamulus listed?
* If the `Jamulus` folder exists, _copy_ it (whole lot) somewhere else and rename it something obvious.
* Then, if Jamulus is listed in "Add or Remove programs", uninstall it.
* Next, in Windows Explorer, go to `%PROGRAMFILES%` and make sure no `Jamulus` folder exists (delete it with Administrator override).
* Similarly, if you're running Windows 64bit, check `%PROGRAMFILES(X86)%` to make sure `Jamulus` doesn't exist there, either.
* Now run the Jamulus 3.8.1 Windows installer again -- make sure it is _not_ the "JACK" one.
* Say "No" to running Jamulus at the end of the install.

My next question is what ASIO device driver are you using?  Have you checked it's working correctly for other applications?  Do that _before_ you first run Jamulus to ensure that's not the problem.

Once you're happy with the ASIO driver, run Jamulus and see if it starts up now.

If it does start, exit Jamulus and go back to `%APPDATA%`, take another backup of the `Jamulus` folder and try restoring the earlier copy you took (if any).

**Reply by Braymanlw** [reply]
Hi Peter,
Thanks for your prompt reply.  Tried your suggestion. Didn't work
I have two interfaces. A PreSonus AudioBox iTwo and a Yamaha AG 06 each with their own ASIO driver and these work fine as does the application on two different laptops.  Interestingly enough if you try to boot up Jamulus with the interface disconnected you normally get an error message.  Tried this with my desktop and still nothing.
Cheers,
Louis.

**Reply by pljones** [reply]
> Didn't work

Given I asked quite a lot of questions, that's not a very helpful response.

**Comment by Braymanlw** [comment]
Hi Henke,Thanks again.  Unfortunatly tried your suggestion but didn't
work.Thanks again.Cheers,Louis

----- Original Message -----

**Reply by ghost** [reply]
Here's a "deep clean" guide (see 7th bullet edit)...
- Open Windows Explorer and enter `%APPDATA%` in the location bar.
- Does a `Jamulus` folder exist inside?
- What, if any, any content is in the `Jamulus` folder?
- If you go into Windows "Add or Remove programs", is Jamulus listed?
- If the `Jamulus` folder exists, copy it (whole lot) somewhere else and rename it something obvious.
- Then, if Jamulus is listed in "Add or Remove programs", uninstall it.
- (see (1) below: For a default new installation; having backed up the aforementioned `Jamulus` folder delete it.)
- Next, in Windows Explorer, go to `%PROGRAMFILES%` and make sure no `Jamulus` folder exists (delete it with Administrator override).
- Similarly, if you're running Windows 64bit, check `%PROGRAMFILES(X86)%` to make sure `Jamulus` doesn't exist there, either.
- Now run the Jamulus 3.8.1 Windows installer again -- make sure it is not the "JACK" one.
- Say "No" to running Jamulus at the end of the install.

(1) Old `jamulus.ini` and `jamulusserver.ini` files remaining in the `%APPDATA%` `Jamulus` folder will affect a new installation.

**Comment by Braymanlw** [comment]
Hi Peter,
Given that my original query (see immediately below) covered most of
your original questions I failed to notice your first three
questions.  For that, I apologise.

I have recently installed Jamulus . When I run it it runs but there is
no graphic interface i.e. nothing on the screen and it does not show
on the Task Bar but is running in Task Manager. Tried running it from
a flash drive which works on a laptop but with same result. Running
Windows 10

	Here's a "deep clean" guide... 

	* Open Windows Explorer and enter %APPDATA% in the location bar.

	* 

	* Does a Jamulus folder exist inside?  HOW ELSE WOULD IT RUN?

	* 
	* What, if any, any content is in the Jamulus folder?  THERE ARE 6
FOLDERS AND 14 FILES IN THIS FOLDER.

	* 

	* If you go into Windows "Add or Remove programs", is Jamulus
listed?  YES.
	* If the Jamulus folder exists, _copy_ it (whole lot) somewhere else
and rename it something obvious.
	* Then, if Jamulus is listed in "Add or Remove programs", uninstall
it.
	* Next, in Windows Explorer, go to %PROGRAMFILES% and make sure no
Jamulus folder exists (delete it with Administrator override).

	* Similarly, if you're running Windows 64bit, check
%PROGRAMFILES(X86)% to make sure Jamulus doesn't exist there,
either.  The Jamulus installer defaults to Program FILE. BUT I HAVE
TRIED BOTH.
	* Now run the Jamulus 3.8.1 Windows installer again -- make sure it
is _not_ the "JACK" one.
	* Say "No" to running Jamulus at the end of the install.

	My next question is what ASIO device driver are you using? Have you
checked it's working correctly for other applications? Do that
_before_ you first run Jamulus to ensure that's not the problem.  I
HAVE TWO DRIVERS FOR TWO AUDIO INTERFACES ALL WORK PERFECTLY ON TWO
OTHER LAPTOPS BUT NOT WITH THE DESKTOP.  I THREE VERSIONS OF JAMULUS
ALL AF WHICH WORK PERFECTLY ON THE LAPTOPS AND NONE OF WHICH WORK ON
THE DESKTOP.  ALL THREE USED TO WORK ON THE DESKTOP BUT NOW IT
DOESN'T.

	THIS INDICATES TO ME THAT THE FAULT LIES IN THE GRAPHICS SET UP IN
WINDOWS 10 ON THE DESKTOP POSSIBLY IN THE REGISTRY BUT IT'S ABOVE MY
PAY GRADE AS TO FIGURE OUT WHY.

	Once you're happy with the ASIO driver, run Jamulus and see if it
starts up now.  NO IT DOESN'T.

	If it does start, exit Jamulus and go back to %APPDATA%, take another
backup of the Jamulus folder and try restoring the earlier copy you
took (if any). 

----- Original Message -----
From: "jamulussoftware/jamulus"
***@***.***>
To:"jamulussoftware/jamulus" 
Cc:"Braymanlw" , "Author" 
Sent:Thu, 11 Nov 2021 10:02:44 -0800
Subject:Re: [jamulussoftware/jamulus] No Graphic interface Windows 10
(Discussion #2117)

	Didn't work  

	Given I asked quite a lot of questions, that's not a very helpful
response. 

	—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub [1], or unsubscribe
[2].
Triage notifications on the go with GitHub Mobile for iOS [3] or
Android [4].  @***@***.******@***.***>

Links:
------
[1]
https://github.com/jamulussoftware/jamulus/discussions/2117#discussioncomment-1627332
[2]
https://github.com/notifications/unsubscribe-auth/AWNCVFBUVA45GX7OD7HBJFLULQAMJANCNFSM5HVWGRPA
[3]
https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&amp;mt=8&amp;pt=524675
[4]
https://play.google.com/store/apps/details?id=com.github.android&amp;referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub

**Reply by pljones** [reply]
> Does a Jamulus folder exist inside?  HOW ELSE WOULD IT RUN?

Jamulus will happily start without it.  (This is `%APPDATA%`, which is not where the program lives.)

> What, if any, any content is in the Jamulus folder?  THERE ARE 6
FOLDERS AND 14 FILES IN THIS FOLDER.

This sounds **very odd**.

![image](https://user-images.githubusercontent.com/1549463/141506957-948a3cd7-f40b-4bac-8c50-6f5abc1ff56e.png)
No folders in my %APPDATA%\Jamulus.  If those are there on a "clean install", there's something else happening other than a "clean install".

You'll notice in my folder a lot of "Something.ini" files -- by default, there should be only two at most.  None of the other files (the .bat and *Settings files) would be there.

Can you confirm what was there immediately after the clean install?
